### DISCUSSION TITLE
How to edit forward and destination in Heroku

### DISCUSSION THREAD

**Discussion by Chappie18** [aahnik/tgcf#217] [Q&A] [answered]
I've deployed tgcf on Heroku, and I am in a private channel and i want to forward live message to my own channel. But i have only private channel's ID , no private link or username.
So how do i set forward command and destination command?
I tired to use like this and its not working.

```
admins: [603520334] 
forwards:
  - source: -1001255537881
  -     dest: [-1001270662904]

show_forwarded_from: false
```

**Comment by ankit-sinha-18** [comment, marked as answer]
This is the config of `TGCF_CONFIG` 👇
https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F#example-configuration

You can use channel id in source and destination id

**Reply by aahnik** [reply]
To set souce and destination, there is no more need to write configuration file. You can use commands.

If you have deployed using a bot account, use the command `/forward` like

```
/forward source: souceID
dest: [dest1,dest2]
```

(you need to send the above command to the bot)

For userbot use `.forward` instead. Send it to Saved Messages.

**Reply by Chappie18** [reply]
I have deployed using user account.So command would be like this right!?

. forward source: -1001255537881
dest: [-1001244437881]

**Reply by aahnik** [reply]
yup, try it!

read the wiki about yaml syntax. very soon web app is coming. then you can set everything intuitively from a GUI

**Reply by ankit-sinha-18** [reply]
> To set souce and destination, there is no more need to write configuration file. You can use commands.

Since there is no more need to write configuration file, Will bot will forget the configuration after bot restart?

**Reply by aahnik** [reply]
yes,

**Reply by aahnik** [reply]
for persistence, i will be introducing connection to GitHub gists in the upcoming release.

**Reply by zoubinho** [reply]
> I have deployed using user account.So command would be like this right!?
> 
> . forward source: -1001255537881
> dest: [-1001244437881]

Hi. I tried to use the config based on the [wiki](https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F) and also tried to amend the config based on this quote, but still my msg can't get forwarded. Anyone has a suggestion? My Heroky worker works fine and logs indicate that my config doesn't have any typo or else...
